style.css

```diff
@@ -0,0 +1,176 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    font-family: "Roboro",sans-serif;
+    background-color:cyan;
+}
+header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 60px;
+    padding: 15px;
+    background-color:cornflowerblue;
+
+}
+main{
+    height: calc(100vh - 70px);
+   
+    display: flex;
+}
+@media(max-width:768px){
+    .side-bar{
+        display: none;
+    }
+}
+.menu{
+    padding: 0 7px;
+    cursor: pointer;
+  
+}
+.search{
+    display: flex;
+}
+.search form{
+    display: flex;
+    border: 1px solid crimson;
+    height: 45px;
+    border-width: 2px;
+
+}
+.search input{
+width: 600px;
+padding: 10px;
+border: 0;
+border-radius: 2px 0 0 2px;
+}
+input:focus{
+    outline: none;
+    border: 1px solid cyan;
+
+}
+.search button{
+    height: 100%;
+    width: 60px;
+    border: none;
+    background-color: bisque;
+}
+.content iframe{
+/* padding-left: 30px; */
+  /* padding-top: 0%; */
+  display: block;
+height: 55%;
+left: 3%;
+position: absolute;
+top: 15%;
+width: 55%;
+
+   
+
+}
```

Comment:
kindly avoid white spaces before closing braces, this will make your.css linters not pass. Kindly avoid any empty white space before closing braces.
Also, stick to 2 spaces before writing inside the braces.